README.md

```diff
@@ -34,9 +34,9 @@ See [Contributing Guide](https://github.com/e18e/e18e/blob/main/CONTRIBUTING.md)
 ## Credits
 
 - The cleanup docs were originally part of [James Garbutt](https://github.com/43081j)'s ecosystem-cleanup.
-- The docs site was inited using the [Vite docs](https://vitejs.dev) as a base.
+- The docs site was created using the [Vite docs](https://vitejs.dev) as a base.
 - The repo follows [Anthony Fu](https://github.com/antfu)'s recommendations.
 
 ## License
 
-[MIT](./LICENSE) License © 2024-Present e18e Contributors
+[MIT](./LICENSE) License, © 2024 to present, e18e contributors
```

Comment:
I don't think we need to change this

Reply:
You need to separate the license from the copyright, either with a newline or a comma.

Reply:
Personally, I would separate with a period instead